example-data/xml-set-1/minimal-00001.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+  <manuscript>
+    <country>Country 1</country>
+    <production-data>
+      <production-data-doi>11.1234/pub.00001</production-data-doi>
+    </production-data>
+    <version>
+      <key>00001|0</key>
+      <abstract>This is quite abstract.</abstract>
+      <author-approval-date>2017-01-01T12:34:56Z</author-approval-date>
+      <decision>Reject Full Submission</decision>
+      <decision-date>2017-02-01T12:13:14Z</decision-date>
+      <ejp-decision>Reject</ejp-decision>
+      <is_resubmission>false</is_resubmission>
+      <manuscript-number>minimal-00001</manuscript-number>
+      <manuscript-type>Research Article</manuscript-type>
+      <submission-date>2017-01-01T12:13:14Z</submission-date>
```

Comment:
This date format does not match with what EJP provide:

`<submission-date>9th May 18  06:28:05</submission-date>`

I know the `csv_generator` handles both formats, but if something else ends up parsing these examples it might be better to have the real format present that they use.

Reply:
Okay, fair enough. eJP did actually generate dates in ISO format (in that format). They seem to have switched to the display format more recently. Actually maybe we should raise it with them to clarify why they changed it what timezone it is in.

Reply:
I sent an email to eJP. There are definitely historic XML files with the ISO datetime format around. Any process will need to handle both formats.

Rather than changing all of the examples I'd suggest to add one XML example with the new datetime. Does that seem sensible?

Reply:
Yer that seems fine. We already handle both, as long as there is mention of it in the examples somewhere I think that's fine.

Reply:
I added `with-display-datetime-format-00014.xml`